tests/rados/test_pg_split.py

```diff
@@ -0,0 +1,87 @@
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from tests.rados.rados_test_util import create_pools, write_to_pools
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.log import Log
+from utility.utils import method_should_succeed, should_not_be_empty
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Automates PG split and merge test scenarios.
+    1. Create replicated pool
+    2.
+    """
+    try:
+        log.info(run.__doc__)
+        config = kw["config"]
+        rhbuild = config.get("rhbuild")
+        cephadm = CephAdmin(cluster=ceph_cluster, **config)
+        rados_obj = RadosOrchestrator(node=cephadm)
+        pool_obj = PoolFunctions(node=cephadm)
+        client_node = ceph_cluster.get_nodes(role="client")[0]
+        timeout = config.get("timeout", 10800)
+
+        log.info("Running PG split merge scenarios")
+        pool = create_pools(config, rados_obj, client_node)
+        should_not_be_empty(pool, "Failed to retrieve pool details")
+        write_to_pools(config, rados_obj, client_node)
+        # Set recover threads configurations
+        if not rhbuild.startswith("6"):
```

Comment:
If the RHCS version is 6, do we plan to change the WPQ profile to `Background recovery ` to increase ` HIGH_RECOVERY_OPS ` ?

Reply:
will do it for 6.x suites